File: src/classes/sq/Daedalus.cpp
#include <NoNut/core/CommonHeader.h>
#include "Daedalus.h"

namespace nonut
{
    Daedalus* Daedalus::get()
	{
		if (inst == nullptr)
		{
			inst = new Daedalus();
		}
		return inst;
	}

	Daedalus::Daedalus() :
		StaticClass("Daedalus"),
        
		METHOD_CTOR(index),
		METHOD_CTOR(symbol),
		METHOD_CTOR(instance)
	{
	}
}

File: g2o/classes/daedalus.py
import sqg2o

class Daedalus:
    
    @staticmethod
    def index(value : str) -> int:
        return sqg2o.Daedalus.index(value)
    
    @staticmethod
    def symbol(value : str) -> dict:
        return sqg2o.Daedalus.symbol(value)
    
    @staticmethod
    def instance(value : str) -> dict:
        return sqg2o.Daedalus.instance(value)